Readme.md

```diff
@@ -11,42 +11,83 @@ on `localhost:3000`
 Example Queries:
 
 ```
-query getNameById($id: String) {
-  profile(id: $id) {
+query getStories {
+  stories {
+    id
     name
+    owner {
+      id
+    }
+    createdAt
   }
 }
 
-query getProfile($id: String) {
-  profile(id: $id) {
+query getNameById($userId: String!) {
+  profile(id: $userId) {
+    name
+  }
+}
+
+query getProfile($userId: String!) {
+  profile(id: $userId) {
+    id
     name
     createdAt
+    updatedAt
+  }
+}
+
+mutation createProfile($userName:String!) {
+  createProfile(input: {name: $userName}) {
+    id
   }
 }
 
-mutation createProfile {
-  createProfile(name: "Andrej") {
+mutation createStory($storyName:String!, $userId:String!) {
+  createStory(
+    input:{
+      name: $storyName,
+      ownerId: $userId
+    }
+  ) {
     id
   }
 }
+
+query schema {
+  __schema {
+    types {
+      name
+      description
+    }
+  }
+}
 ```
 
 Query Variables: (needs correct ID)
 ```
 {
-  "id": "9UFFp7veQ2AH-lbsgu7dg"
+  "userName": "Felix",
+  "userId": "N9klwfiU_3DySdUFI8DZP",
+  "storyName": "Waterfall"
 }
 ```
 
-## Check Data 
+1. Run "createProfile"
+2. Copy `id`to Query Variables `userId`
+3. Run "createStory"
+4. Run "getStories"
 
-- file system: `ls -la ~/.storylock/`
+Repeat Step 3 and change `storyName` and / or `userId` to create more stories.
 
-# Usage with IPFS
-todo
 
+## Check Data
 
-# Download schema
+- file system: `ls -la ~/.storyloc/`
+
+Remove this folder to clean data.
+
+# Download schema for Clients
 
 To download the schema install Apollo CLI.
 On macOS use `brew install apollo-cli`, there are some fixed issued included. 
```

Comment:
can we [link](https://github.com/apollographql/apollo-tooling)

Reply:
yes